src/lib/EventPublisher.js

```diff
@@ -59,6 +74,11 @@ class EventPublisher extends EventEmitter {
 
     this._emitWithPrefix(eventData.name, event);
     this.emit(ALL_EVENTS, event);
+
+    // broadcast events only from the current worker
+    if (!event.fromMaster && cluster.isWorker) {
```

Comment:
Why not do this
1. worker 1: get event from device
2. worker 1: in webhookManager(1), get the event and do webhook job
3. worker 1: send it to master (if we swap out worker 1 for master in steps 1/2 we can ignore this one)
4. master: get the event and broadcast it to all other workers
5. master + worker 1,2,3,4: get the event from master and publish it to itself only, we prevent mirroring with 
6. Since all this code is running in event manager, it calls `_publish` and doesn't see the logic for sending the event to master or calling the webhook
7. master + worker 1,2,3,4: They will never run code that calls webhook manager so we don't need to filter